File: testdata/repos/vbnetsample/Forms/MainForm.vb
Imports System.Windows.Forms
Namespace Demo.Forms
Public Partial Class MainForm
    Public Event Authorized As EventHandler
    Public Property CurrentUser As String
    Private Sub Submit_Click(sender As Object, e As EventArgs) Handles SubmitButton.Click
        RaiseEvent Authorized(Me, e)
    End Sub
End Class
End Namespace

File: testdata/repos/dotnetsample/Forms/MainForm.cs
using System;
using System.Windows.Forms;

namespace DotNetSample.Forms;

public partial class MainForm : Form
{
    public MainForm()
    {
        InitializeComponent();
    }

    private void MainForm_Load(object sender, EventArgs e)
    {
        Text = "Ready";
    }
}
